services/decoder/protocols/wormhole/wormhole.configs.ts

```diff
@@ -0,0 +1,144 @@
+        chain_name: "avalanche-mainnet",
+    },
+    {
+        address: "0x54a8e5f9c4CbA08F9943965859F6c34eAF03E26c",
+        is_factory: false,
+        protocol_name: "wormhole",
+        chain_name: "avalanche-mainnet",
+    },
+    {
+        address: "0x5848C791e09901b40A9Ef749f2a6735b418d7564",
+        is_factory: false,
+        protocol_name: "wormhole",
+        chain_name: "emerald-paratime-mainnet",
+    },
+    {
+        address: "0xfE8cD454b4A1CA468B57D79c0cc77Ef5B6f64585",
+        is_factory: false,
+        protocol_name: "wormhole",
+        chain_name: "emerald-paratime-mainnet",
+    },
+    {
+        address: "0xa321448d90d4e5b0A732867c18eA198e75CAC48E",
+        is_factory: false,
+        protocol_name: "wormhole",
+        chain_name: "aurora-mainnet",
+    },
+    {
+        address: "0x51b5123a7b0F9b2bA265f9c4C8de7D78D52f510F",
+        is_factory: false,
+        protocol_name: "wormhole",
+        chain_name: "aurora-mainnet",
+    },
+    {
+        address: "0x796Dff6D74F3E27060B71255Fe517BFb23C93eed",
+        is_factory: false,
+        protocol_name: "wormhole",
+        chain_name: "celo-mainnet",
+    },
+    {
+        address: "0xa321448d90d4e5b0A732867c18eA198e75CAC48E",
+        is_factory: false,
+        protocol_name: "wormhole",
+        chain_name: "celo-mainnet",
+    },
+    {
+        address: "0xb1731c586ca89a23809861c6103f0b96b3f57d92",
+        is_factory: false,
+        protocol_name: "wormhole",
+        chain_name: "moonbeam-mainnet",
+    },
+    {
+        address: "0xC8e2b0cD52Cf01b0Ce87d389Daa3d414d4cE29f3",
+        is_factory: false,
+        protocol_name: "wormhole",
+        chain_name: "moonbeam-mainnet",
+    },
+    {
+        address: "0xa5f208e072434bC67592E4C49C1B991BA79BCA46",
+        is_factory: false,
+        protocol_name: "wormhole",
+        chain_name: "arbitrum-mainnet",
+    },
+    {
+        address: "0x0b2402144Bb366A632D14B83F244D2e0e21bD39c",
+        is_factory: false,
+        protocol_name: "wormhole",
+        chain_name: "arbitrum-mainnet",
+    },
+    {
+        address: "0x1D68124e65faFC907325e3EDbF8c4d84499DAa8b",
+        is_factory: false,
+        protocol_name: "wormhole",
+        chain_name: "optimism-mainnet",
+    },
+    {
+        address: "0xEe91C335eab126dF5fDB3797EA9d6aD93aeC9722",
+        is_factory: false,
+        protocol_name: "wormhole",
+        chain_name: "optimism-mainnet",
+    },
+    {
+        address: "0xa321448d90d4e5b0A732867c18eA198e75CAC48E",
+        is_factory: false,
+        protocol_name: "wormhole",
+        chain_name: "gnosis-mainnet",
+    },
+    {
+        address: "0x8d2de8d2f73F1F4cAB472AC9A881C9b123C79627",
+        is_factory: false,
+        protocol_name: "wormhole",
+        chain_name: "base-mainnet",
+    },
+    {
+        address: "0xbebdb6C8ddC678FfA9f8748f85C815C556Dd8ac6",
+        is_factory: false,
+        protocol_name: "wormhole",
+        chain_name: "base-mainnet",
+    },
```

Comment:
Some of the configs have the same chain twice (base, optimism, moonbeam etc)
Is that intended?

Reply:
There are 2 different contracts, Wormhole Core and Wormhole Token Bridge